packages/react/dismissable-layer/src/DismissableLayer.stories.tsx

```diff
@@ -529,7 +529,7 @@ function DummyDialog({ children, openLabel = 'Open', closeLabel = 'Close' }: Dum
                     {(focusScopeProps) => (
                       <div
                         {...dismissableLayerProps}
-                        ref={composeRefs(dismissableLayerProps.ref, focusScopeProps.ref)}
```

Comment:
I believe `composeRefs` probably isn't used in the file anymore.

Reply:
You believe correct!